src/App.js

```diff
@@ -30,6 +31,8 @@ function App() {
   return (
     <ThemeProvider theme={paisanosTheme}>
       <main>
+
+        <Route exact path='/webHome' render={(props) => <WebHome {...props} />} />
         <Route exact path='/home' render={() => <Home />} />
```

Comment:
Also, I think we can replace actually home component, what do you think?

Reply:
done, I did already the changes :)